aten/src/ATen/core/dispatch/Dispatcher.cpp

```diff
@@ -147,7 +147,7 @@ void Dispatcher::deregisterLibrary_(const std::string& ns) {
   libraries_.erase(ns);
 }
 
-RegistrationHandleRAII Dispatcher::registerDef(FunctionSchema schema, std::string debug) {
+RegistrationHandleRAII Dispatcher::registerDef(FunctionSchema schema, std::string debug, const std::vector<at::Tag>& tags) {
```

Comment:
Taking a const ref to vector is suboptimal here. As you need an owning reference accepting the vector by value will avoid an unnecessary copy when the argument is a temporary.